ui/src/app/cohort-review/participant-annotation/participant-annotation.component.ts

```diff
@@ -14,6 +14,8 @@ import {
   ParticipantCohortAnnotation as Annotation,
 } from 'generated';
 
+const CDR_VERSION = 1;
```

Comment:
Should we start passing this around on the client?

If not, add a TODO so we don't forget to change this later...

Reply:
I haven't made a ticket for it yet, but pretty soon I'll have a PR where the client uses the actual CDR version everywhere its supposed to.  Right now we're using `CDR_VERSION` all over as a placeholder, so a `grep -R "CDR_VERSION" **/*.ts` should yield every instance where the code needs to be updated.

@freemabd1 you can put `// TODO (jms)` as well, since I'll be handling it down the line.

Reply:
For tracking: https://precisionmedicineinitiative.atlassian.net/browse/RW-376?jql=assignee%20%3D%20currentUser()%20AND%20resolution%20%3D%20unresolved%20ORDER%20BY%20priority%20DESC%2C%20created%20ASC

Reply:
I added TODO to each occurrence.